src/IECoreGL/ShaderStateComponent.cpp

```diff
@@ -43,12 +43,19 @@
 #include "IECore/MessageHandler.h"
 #include "IECore/SplineData.h"
 
+#include "boost/algorithm/string.hpp"
+
 #include <iostream>
 
 using namespace std;
 using namespace IECore;
 using namespace IECoreGL;
 
+namespace
+{
+	const InternedString g_maxTextureResolutionParameterSuffix( "_maxResolution" );
```

Comment:
Can we use ":maxResolution" here - that feels like it better matches are general conventions for namespacing things. Since ":" isn't a valid character for a true GLSL shader parameter, I think it eliminates any possibility of a clash.

Also, don't indent for namespaces.

Reply:
Done in a7d166d